game/screen_intro.cc

```diff
@@ -183,13 +184,13 @@ void ScreenIntro::populateMenu() {
 			for (auto const& item: submenu.items) {
 				ConfigItem& c = config[item];
 				opts.emplace_back(MenuOption(c.getShortDesc(), c.getLongDesc()));
-                                opts.back().changer(c);
+								opts.back().changer(c);
 			}
 			configmain.emplace_back(MenuOption(submenu.shortDesc, submenu.longDesc, imgConfig));
-                        configmain.back().submenu(std::move(opts));
+						configmain.back().submenu(std::move(opts));
```

Comment:
Not quite sure why this spacing is needed? (might be my browser)
Allign it with configmain